pkg/openstack/galera.go

```diff
@@ -64,7 +68,7 @@ func ReconcileGaleras(
 			// at the initial deployment because there is no SST involved when the DB is bootstrapped
 			// as there are no data to be transferred yet.
 			Subject: &certmgrv1.X509Subject{
-				Organizations: []string{fmt.Sprintf("%s.cluster.local", instance.Namespace)},
+				Organizations: []string{fmt.Sprintf("%s.%s", instance.Namespace, ClusterInternalDomain)},
```

Comment:
this seems like an unrelated change.

Reply:
yes, it was a drive by to use the const we have. I can to a separate PR, but should be trivial enough to keep it?

Reply:
OK to keep it.